Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -576,15 +576,28 @@ public extension Purchases {
     func logOut() async throws -> CustomerInfo {
         return try await logOutAsync()
     }
+
     @objc func getOfferings(completion: @escaping (Offerings?, PublicError?) -> Void) {
-        self.offeringsManager.offerings(appUserID: appUserID) { result in
+        self.getOfferings(fetchPolicy: .default, completion: completion)
+    }
+
+    internal func getOfferings(
```

Comment:
Looks like we are not testing these new methods directly. This is called by the existing `getOfferings` method, so it's tested with a `default` fetch policy but since it's a public method, even if it's internal, I think we should add a quick test for this just in case.